run_dev_backend.sh

```diff
@@ -126,6 +126,8 @@ if [ "$mode" = "cypress" ]; then
     tracimcli db delete --force || true
     tracimcli db init || true
     cp /tmp/${DATABASE_NAME}.sqlite /tmp/${DATABASE_NAME}.sqlite.tmp
+    export TRACIM_TRANSLATION_SERVICE__ENABLED=True
+    export TRACIM_TRANSLATION_SERVICE__PROVIDER=test
```

Comment:
This `test`is a forgot debug?

Reply:
No, this is needed to configure a mock/test-only translation service in the backend so that the cypress tests is functional.
Without it we cannot test the translation in frontend as the translate button is only displayed when the backend has a configured translation service.